stack/stack_custom.yaml

```diff
@@ -39,7 +39,7 @@ udunits:
 hdf5:
   build: YES
   version: 1.10.6
-  shared: YES
```

Comment:
This is the line causing the CI to fail. I think this should be kept as `shared: YES`. We use `stack_NOAA.yaml` for static libraries with `shared: NO`. On user systems shared libraries are ok, but UFS likes static libraries on HPC systems.

Reply:
Restored shared libraries for custom and Mac stacks (fd8237c9a467d55de02a9440c7538096bf806baa)